server/plugin.go

```diff
@@ -41,6 +42,10 @@ func (p *Plugin) OnActivate() error {
 	}
 
 	split := strings.Split(p.configuration.TeamChannel, ",")
+	if len(split) != 2 {
+		return errors.New("teamChannel setting doesn't follow the pattern $TEAM_NAME,$CHANNEL_NAME")
```

Comment:
is it worth printing the `p.configuration.TeamChannel` variable so the person knows exactly where the mismatch is? for better forensics and better debugging. Just a thought.

Reply:
I don't really see the value. The admin has to go to the admin console any way, where he/she will see the value.